std/assembly/util/memory.ts

```diff
@@ -159,32 +152,36 @@ export function memmove(dest: usize, src: usize, n: usize): void { // see: musl/
     }
   }
   if (dest < src) {
-    if ((src & 7) == (dest & 7)) {
-      while (dest & 7) {
-        if (!n) return;
-        --n;
-        store<u8>(dest++, load<u8>(src++));
-      }
-      while (n >= 8) {
-        store<u64>(dest, load<u64>(src));
-        n    -= 8;
-        dest += 8;
-        src  += 8;
+    if (ASC_SHRINK_LEVEL < 1) {
```

Comment:
What do you think of gating these behind `< 2`? It's not a lot of code, yet is much faster.

Reply:
Also thinking about this